app/src/main/java/org/apache/syncope/client/enduser/pages/MustChangePassword.java

```diff
@@ -53,8 +52,8 @@ protected void doPwdSubmit(final AjaxRequestTarget target, final AjaxPasswordFie
             parameters.add(Constants.NOTIFICATION_TITLE_PARAM, getString("self.pwd.change.error"));
             parameters.add(Constants.NOTIFICATION_MSG_PARAM, getString("self.pwd.change.error.msg"));
             SyncopeEnduserSession.get().onException(e);
-            notificationPanel.refresh(target);
         }
+        notificationPanel.refresh(target);
```

Comment:
dobbiamo decidere se vogliamo tenere notification o andare sulla pagina di result

Reply:
Redirecting to error page could not often be very helpful for the user. For example while changing password if it is not compliant with password policy it would be useful to notify an error to the user and remain in the same page in order to insert a compliant password.